doc/reference/command.rst

```diff
@@ -0,0 +1,75 @@
+Commands
+=============
+
+
+.. This document is generated via the `development:command-reference` command
+
+
+.. contents::
+   :depth: 2
+   :backlinks: none
+   :local:
+
+
+.. _GotoDefinitionHandler:
+
+
+GotoDefinitionHandler
+---------------------
+
+
+.. _param_language:
```

Comment:
needs to include `_RpcCommand_GlobalDefinitionHandler_language` or similar as these are global references

Reply:
Do we have an example of non global handlers?